plugins/inputs/apache/apache.go

```diff
@@ -61,21 +84,47 @@ func (n *Apache) Gather(acc telegraf.Accumulator) error {
 	return outerr
 }
 
-var tr = &http.Transport{
-	ResponseHeaderTimeout: time.Duration(3 * time.Second),
-}
+func (n *Apache) gatherUrl(addr *url.URL, acc telegraf.Accumulator) error {
+	var err error
+	var resp *http.Response
+	var req *http.Request
+	var tr *http.Transport
+	var client *http.Client
+
+	if addr.Scheme == "https" {
+		//fmt.Printf("SSL skip verify = true\n")
+		tr = &http.Transport{
+			ResponseHeaderTimeout: time.Duration(3 * time.Second),
```

Comment:
I don't think so, ResponseHeaderTimeout is different than Client Timeout  as you can see in https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/ and I've hardcoded as in the original apache input plugin to 3 seconds. 
If you think these two different Timeouts could be controlled by the same input config , I will change in the next commit ok?